fbchat/client.py

```diff
@@ -469,14 +474,25 @@ def resetDefaultThread(self):
     END DEFAULT THREAD METHODS
     """ 
     def removeFriend(self, friend_id=None):
-        """param friend_id: The id of the friend that you want to remove"""
+        """
+        Removes a specifed friend from your friend list
+        :param friend_id: The id of the friend that you want to remove
+        :returns error when the removing was unsuccessful
+        :returns True when the removing was successful
```

Comment:
The syntax in the docstrings is quite important, since it gets executed and published on [RTFD](https://fbchat.readthedocs.io). `:returns` is wrong, it should be `:return:`, and there should only be one of them